samples/subsys/lorawan/fuota/README.rst

```diff
@@ -0,0 +1,37 @@
+.. zephyr:code-sample:: lorawan-fuota
+   :name: LoRaWAN FUOTA
+   :relevant-api: lorawan_api
+
+   Demonstration of LoRaWAN firmware-upgrade over the air (FUOTA).
```

Comment:
This should be worded as if you were completing the sentence "This code sample shows how to ..."
ex:

```suggestion
   Perform a LoRaWAN firmware-upgrade over the air (FUOTA) operation.
```